### PR TITLE
Bugfixes

### LINKED ISSUE #108: A user can modify a Task's project_id, thus bypassing security contraints
Certain fields in a Task should not be editable, such as the project field, which allows a user to reassign a task to a project he/she does not own or has permission to access.

### PR DESCRIPTION
**Pull request by pierotofy**
Fixes #108  #115 #80 #110 #107.

@dakotabenjamin I plan to merge this by tomorrow, since #115 is quite critical. Let me know if you find anything that should be changed.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -419,6 +419,18 @@ def get_tile_json_data(self):
             }
         }
 
+    def get_textured_mesh_archive(self):
+        archive_path = self.assets_path("odm_texturing.zip")
+        textured_mesh_directory = self.assets_path("odm_texturing")
```
Comment (dakotabenjamin) [inline app/models.py]: I don't know what self.assets_path refers to, but you might not want to hard-code the name here

Comment (pierotofy) [inline app/models.py]: Yes, I've now put it into a separate method "get_textured_mesh_filename".

Comment (dakotabenjamin) [inline app/models.py]: :+1:

**Comment by pierotofy** [comment]
Thanks @dakotabenjamin !

**Pull request merged**